src/compiler/checker.ts

```diff
@@ -30336,6 +30476,13 @@ namespace ts {
             return false;
         }

```

Comment:
In `checkAmbientInitializer` you need to `isInvalidInitializer` also need to be false if `node.initializer` is `BigIntLiteral`

Reply:
And `PrefixUnary` with bigint literal. (note you dont want to change `isStringOrNumberLiteralExpression` since its used for enum member access as well but add one similar to that)

Reply:
I believe `isStringOrNumberLiteralExpression(node.initializer) || isSimpleLiteralEnumReference(node.initializer) || node.initializer.kind === SyntaxKind.TrueKeyword || node.initializer.kind === SyntaxKind.FalseKeyword` will already be false if `node.initializer.kind === SyntaxKind.BigIntLiteral`. Am I wrong?

Reply:
I meant it should be true if node.initializer is bigIntLiteralExpression. (we want to retain `const x = 12n` in dts just like `const x = 12`) Add that as a test as well.